package.json

```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "codecov": "^3.7.0",
     "compression-webpack-plugin": "^4.0.0",
+    "fs-vacuum": "^1.2.10",
```

Comment:
@geekyahmed Do we need this dependency on fs-vacuum? Where is it used within Ambianic UI?  

The [docs state](https://www.npmjs.com/package/fs-vacuum) its purpose is to:
"Remove the empty branches of a directory tree, optionally up to (but not including) a specified base directory. Optionally nukes the leaf directory."